demo/molecule/selectPopover/demo/index.js

```diff
@@ -79,17 +80,32 @@ const Demo = () => {
         ))}
       </MoleculeSelect>
       <br />
-      <input
-        type="checkbox"
-        checked={hasEvents}
-        onChange={ev => setHasEvents(ev.target.checked)}
-      />
-      <label>With events (onOpen & onClose)</label>
+      <div>
+        <label>
+          <input
+            type="checkbox"
+            checked={hasEvents}
+            onChange={ev => setHasEvents(ev.target.checked)}
+          />
+          With events (onOpen & onClose)
+        </label>
+      </div>
+      <div>
+        <label>
+          <input
+            type="checkbox"
+            checked={actionsAreHidden}
+            onChange={ev => setActionsAreHidden(ev.target.checked)}
+          />
+          Actions are hidden
+        </label>
+      </div>
```

Comment:
👏🏼

Reply:
Thanks for the recommendation @nucliweb 😄